nixnet/_ctypedefs.py

```diff
@@ -105,7 +105,7 @@ class nxDatabaseRef_t(u32):  # NOQA: N801
     pass
 
 
-class nxStatus_t(i32):  # NOQA: N801
+class nxStatus_t(u32):  # NOQA: N801
```

Comment:
Please document the reason for this in your commit message.

Well, since its more of a "I told you to", I guess I'll give a write-up
> Change `nxStatus_t` to be a `u32` instead of `i32`.  The problem is our constants are defined as unsigned while `nxStatus_t` was defined as signed.  This caused several problems (incorrect detection of errors, failure to convert code to enum value).
> For now, we are going with the "quickest to get working" solution.  In the long term, we need to decide if we should treat these as unsigned or signed.